ui/src/containers/HelpContainer.jsx

```diff
@@ -1,191 +1,177 @@
-import React from 'react';
-import { connect } from 'react-redux';
-import { Container, Card, Row, Col, Button, Form } from 'react-bootstrap';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  Container,
+  Card,
+  Row,
+  Col,
+  Button,
+  Form,
+  Alert,
+} from 'react-bootstrap';
+import { Controller, useForm } from 'react-hook-form';
 
 import { sendFeedback } from '../api/login';
 
 import characterisation from '../help_videos/mx3-characterisation.ogv';
 import interleaved from '../help_videos/mx3-interleaved.ogv';
 import mesh from '../help_videos/mx3-mesh.ogv';
 
-export class HelpContainer extends React.Component {
-  constructor(props) {
-    super(props);
-    this.handleSubmit = this.handleSubmit.bind(this);
-    this.sender = '';
-    this.content = '';
-  }
-
-  handleSubmit() {
-    sendFeedback(this.sender.value, this.content.value);
-    this.sender.value = '';
-    this.content.value = '';
-  }
+const DEFAULT_HELPFORM_VALUES = { email: '', content: '' };
 
-  localContactPanel() {
-    let panel = null;
+function HelpContainer() {
+  const {
+    control,
+    handleSubmit: makeOnSubmit,
+    reset,
+    formState: { errors },
+  } = useForm({ defaultValues: DEFAULT_HELPFORM_VALUES });
+  const user = useSelector((state) => state.login.user);
+  const serverVersion = useSelector((state) => state.general.serverVersion);
 
-    if (this.props.login.user) {
-      const familyName = this.props.login.user.nickname || '';
-      const givenName = this.props.login.user.nickname || '';
-      const email = this.props.login.user.email || '';
-      const tel = '';
+  const [feedback, setFeedback] = useState({});
 
-      panel = (
-        <Card className="mb-3">
-          <Card.Header>
-            <span>Local Contact</span>
-            <span className="me-3 position-absolute end-0">
-              <i className="fas fa-user" />
-            </span>
-          </Card.Header>
-          <Card.Body>
-            <span>
-              Name: {givenName} {familyName}
-              <br />
-              Email: {email}
-              <br />
-              Tel: {tel} <br />
-            </span>
-          </Card.Body>
-        </Card>
-      );
+  async function handleSubmit(data) {
+    try {
+      await sendFeedback(data.email, data.content);
+      setFeedback({
+        variant: 'success',
+        message: 'Feedback sent successfully',
+      });
```

Comment:
The form state has an `isSubmitSuccessful` flag that tells you if the submit was successful (i.e. no error thrown in `handleSubmit`). And if `sendFeedback` throws an error, you can catch it and call the form's `setError` function as explained here: https://react-hook-form.com/docs/useform/seterror

This means that you don't need the `feedback` state and can just declare different alerts in the JSX based on `isSubmitSuccessful` and `errors`.